mgmtsystem_claim/mgmtsystem_claim.xml

```diff
@@ -7,7 +7,9 @@
 		<field name="name">Management System Claims Tree</field>
 		<field name="model">mgmtsystem.claim</field>
 		<field name="arch" type="xml">
-			<tree string="Claims" colors="blue:state=='pending' and not(date_deadline and (date_deadline &lt; current_date));gray:state in ('close', 'cancel');red:date_deadline and (date_deadline &lt; current_date)">
+			<tree string="Claims" >
+				<!-- TODO use stage_id to change color -->
```

Comment:
Is this coment to say that if you want colors, uncomment the line bellow?

If that is the case, it should be worded appropriately, `TODO` gives the impression that there is dev work to be done.